contracts/NFTVesting/TestNFT/BasicNft.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import "hardhat/console.sol";
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+
+
+contract BasicNft is ERC721 {
+    string public constant TOKEN_URI =
+        "ipfs://bafybeig37ioir76s7mg5oobetncojcm3c3hxasyd4rvid4jqhy4gkaheg4/?filename=0-PUG.json";
+    uint256 private s_tokenCounter; // ID of NFT
```

Comment:
Curious why did you name this variable "s_tokenCounter"?

Reply:
https://ethereum.stackexchange.com/questions/127544/what-is-the-naming-convention-around-s-in-solidity

My rationale was it was the main storage variable and care should be taken with its usage